radix-engine-interface/src/blueprints/package/invocations.rs

```diff
@@ -94,8 +94,22 @@ pub struct PackageClaimRoyaltiesInput {}
 pub type PackageClaimRoyaltiesOutput = Bucket;
 
 #[derive(Debug, Clone, Eq, PartialEq, Default, ScryptoSbor, ManifestSbor)]
-pub struct PackageDefinition {
-    pub schema: PackageSchema,
-    pub function_access_rules: BTreeMap<String, BTreeMap<String, AccessRule>>,
-    pub royalty_config: BTreeMap<String, RoyaltyConfig>,
+pub struct PackageSetup {
+    pub blueprints: BTreeMap<String, BlueprintSetup>,
+}
+
+#[derive(Debug, Clone, Eq, PartialEq, Default, ScryptoSbor, ManifestSbor)]
+pub struct BlueprintSetup {
+    pub schema: BlueprintSchema,
+
+    pub function_auth: BTreeMap<String, AccessRule>,
+    pub royalty_config: RoyaltyConfig,
+
+    pub template: BlueprintTemplate,
```

Comment:
Suggest `method_auth_template: MethodAuthTemplate`

Reply:
Will keep this for now as I have a separate PR which updates all naming.